tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationAttachmentPointTest.java

```diff
@@ -29,15 +29,15 @@
  *
  * @since 1.0
  */
-@Test(groups = { "brokenOnNewParser" })
+@Test(groups = { "disableOnOldParser" })
 public class AnnotationAttachmentPointTest {
 
     private CompileResult compileResult;
 
     @BeforeClass
     public void setup() {
         compileResult = BCompileUtil.compile("test-src/annotations/annot_attachments_negative.bal");
-        Assert.assertEquals(compileResult.getErrorCount(), 244);
+        Assert.assertEquals(compileResult.getErrorCount(), 245);
```

Comment:
So theres an additional error being logged - but I don't see any additional assert being added. Should we add that missing one too?

Reply:
yea I ll add that additional assert, in the same position 2 errors are being thrown one by parser and one by sem Analyzer